txt2html.py

```diff
@@ -205,6 +206,18 @@ def main():
     if args.lang:
         lang_attribute_value = args.lang
 
+    if args.config:
+        config_file_path = args.config
+        try:
+            with open(config_file_path, "rb") as config_file:
+                config_data = tomli.load(config_file)
+
+                lang_attribute_value = config_data.get("lang", lang_attribute_value)
```

Comment:
Can you also make the config_data get the output settings?

Reply:
Implemented. Could you please check and confirm if anything else is required?